src/webgpu/api/operation/resource_init/texture_zero_init.spec.ts

```diff
@@ -81,91 +87,49 @@ export const enum InitializedState {
   Zero, // We check that uninitialized subresources are in this state when read back.
 }
 
-export function initializedStateAsFloat(state: InitializedState): number {
-  switch (state) {
-    case InitializedState.Zero:
-      return 0;
-    case InitializedState.Canary:
-      return 1;
-    default:
-      unreachable();
-  }
-}
+const initializedStateAsFloat = {
+  [InitializedState.Zero]: 0,
+  [InitializedState.Canary]: 1,
+};
 
-export function initializedStateAsUint(state: InitializedState): number {
-  switch (state) {
-    case InitializedState.Zero:
-      return 0;
-    case InitializedState.Canary:
-      return 255;
-    default:
-      unreachable();
-  }
-}
+const initializedStateAsUint = {
+  [InitializedState.Zero]: 0,
+  [InitializedState.Canary]: 1,
+};
 
-export function initializedStateAsSint(state: InitializedState): number {
-  switch (state) {
-    case InitializedState.Zero:
-      return 0;
-    case InitializedState.Canary:
-      return -1;
-    default:
-      unreachable();
-  }
-}
+const initializedStateAsSint = {
+  [InitializedState.Zero]: 0,
+  [InitializedState.Canary]: -1,
+};
 
-export function initializedStateAsColor(
+function initializedStateAsColor(
```

Comment:
I think this function could use the textureComponentType of the format instead of matching on substring?

Reply:
It could, but that's in the other patch. I'll fix this up after landing both.